### PR TITLE
Improve the wording about where to download latest (stable) vast release

### PR DESCRIPTION
**Pull request by dispanser**
Wording update for readme.

### :memo: Checklist

- [X] All user-facing changes have changelog entries.
- [X] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [X] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Check that the links work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,9 +55,10 @@ The network telemetry engine for data-driven security investigations.
 
 ## Get VAST
 
-Linux users can download our [latest static binary
-release](https://storage.googleapis.com/tenzir-public-data/vast-static-builds/vast-static-latest.tar.gz)
-via browser or cURL.
+Linux users can download our [latest static
+build](https://storage.googleapis.com/tenzir-public-data/vast-static-builds/vast-static-latest.tar.gz)
+based on maser via browser or cURL. The latest stable version can be found on our
+[releases][release-url] page.
```
Comment (mavam) [inline README.md:61]:
```suggestion
We offer pre-packaged versions of VAST for download:
- **stable*: see the artifacts of the latest official [VAST release][latest-release] 
- **development**: we offer a [static
build](https://storage.googleapis.com/tenzir-public-data/vast-static-builds/vast-static-latest.tar.gz) of the master branch for Linux
```

File: README.md [hunk 2]
```diff
@@ -155,6 +156,7 @@ proceedings][nsdi-proceedings].
 [contributing-url]: https://github.com/tenzir/.github/blob/master/contributing.md
 [since-release-badge]: https://img.shields.io/github/commits-since/tenzir/vast/latest.svg?color=green
 [since-release-url]: https://github.com/tenzir/vast/compare/2021.06.24...master
+[release-url]: https://github.com/tenzir/vast/releases
```
Comment (mavam) [inline README.md:159]:
```suggestion
[latest-release]: https://github.com/tenzir/vast/releases/latest
```

**Comment by dominiklohmann** [comment]
@dispanser I think this can safely be merged regardless of the feature freeze for the release.

File: README.md [hunk 3]
```diff
@@ -55,9 +55,10 @@ The network telemetry engine for data-driven security investigations.
 
 ## Get VAST
 
-Linux users can download our [latest static binary
-release](https://storage.googleapis.com/tenzir-public-data/vast-static-builds/vast-static-latest.tar.gz)
-via browser or cURL.
+We offer pre-packaged versions of VAST for download:
+- **stable*: see the artifacts of the latest official [VAST release][latest-release]
```
Comment (dominiklohmann) [inline README.md:59]:
```suggestion
- **stable**: see the artifacts of the latest official [VAST release][latest-release] 
```

**Review by dominiklohmann — APPROVED** [review]
Fine by me, except for one missing character.

**Pull request merged**